runtime/kusama/src/lib.rs

```diff
@@ -2007,6 +2007,16 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_nomination_pools_rpc_runtime_api::NominationPoolsApi<
+		Block,
+		AccountId,
+		Balance,
+	> for Runtime {
+		fn pending_rewards(member: AccountId) -> Balance {
+			NominationPools::pending_rewards(member)
```

Comment:
where is this function exactly?

Reply:
Will add it.

Reply:
Or should I just put the logic in there?